src/main/java/org/assertj/core/api/AbstractCharSequenceAssert.java

```diff
@@ -174,18 +231,15 @@ public SELF isBlank() {
    * @throws AssertionError if the actual {@code CharSequence} is blank.
    * @since 2.6.0 / 3.6.0
    */
-  public SELF isNotBlank() {
-    strings.assertNotBlank(info, actual);
+  public SELF containsNotOnlyWhitespaces() {
```

Comment:
should be `doesNotContainOnlyWhitespaces`

Reply:
Done